tests/system/conftest.py

```diff
@@ -537,6 +537,33 @@ def penguins_df_default_index(
     return session.read_gbq(penguins_table_id)
 
 
+@pytest.fixture(scope="session")
+def llm_fine_tune_df_default_index(
+    session: bigframes.Session,
+) -> bigframes.dataframe.DataFrame:
+    sql = """
```

Comment:
You can just read_pandas to get bigframes DF.

If not used by other tests, no need to put in conftest.py. Just put in test files. Our conftest is already too large.

Reply:
Done.

Reply:
?